eng/testing/linker/trimmingTests.targets

```diff
@@ -96,6 +96,7 @@
                                                  .Replace('{MonoProjectRoot}', '$(MonoProjectRoot)')
                                                  .Replace('{MonoAOTCompilerTasksAssemblyPath}', '$(MonoAOTCompilerTasksAssemblyPath)')
                                                  .Replace('{JsonToItemsTaskFactoryTasksAssemblyPath}', '$(JsonToItemsTaskFactoryTasksAssemblyPath)')
+                                                 .Replace('{RuntimeConfigParserTasksAssemblyPath}', '$(RuntimeConfigParserTasksAssemblyPath)')
```

Comment:
Similar line needed for `RuntimeConfigParserTaskDir`?

Reply:
CI seems to be working well without it so far. If there is a problem, I will add it.